client/src/ui/components/trading/MarketOrderPanel.tsx

```
@@ -226,469 +234,466 @@
+    } = useDojo();
+    useEffect(() => {
+      setBid(String(lords / resource));
+    }, [resource, lords]);
+
+    const updateLords = useCallback((newBid: number, newResource: number) => {
+      setLords(Number(newBid * newResource));
+    }, []);
+
+    const handleBidChange = (newBid: number) => {
+      const numericBid = Number(newBid);
+      if (!isNaN(numericBid) && numericBid > 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)